assets/charts/control-plane/prometheus-operator-crds/Chart.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: prometheus-operator-crds
+description: A Helm chart for Prometheus Operator CRDS
+type: application
+version: 0.1.0
+appVersion: "0.1.0"
```

It would be nice if we follow consistency. 

```suggestion
appVersion: 0.1.0
```